config/application.yml.example

```diff
@@ -24,6 +24,7 @@ EMAIL_SERVER_NAME: "smtp.something.ca"
 EMAIL_DOMAIN: "something.ca"
 EMAIL_USERNAME: "someuser"
 EMAIL_PASSWORD: "somepassword"
+GRAPHPASS: "/path/to/graphpass/utilty"
```

Comment:
Let's make it slightly more clearer

`GRAPHPASS: "/path/to/graphpass/directory/graphpass`

So they know they have to provide the final `graphpass`

Reply:
That fixed it for me too!  Thanks Ian!